service/src/main/java/io/camunda/service/search/filter/UserTaskFilter.java

```diff
@@ -165,11 +159,11 @@ public Builder processInstanceKeys(final List<Long> values) {
       return this;
     }
 
-    public Builder processDefinitionKeys(final String value, final String... values) {
+    public Builder processDefinitionKeys(final Long value, final Long... values) {
```

Comment:
🔧 I think I had a remark for Roman when we added the process instance filter: This method signature is not very common and also superficial if you ask me. The method signature `final Long... values` itself should be equivalent in behavior and is way more intuitive and common.

Reply:
Sorry @tmetzke  it is not clear  for me.

What action I should take over here? Keep as a String or follow as Long?
Once its keys I think we should proceed with Long value to keep consistency with other keys. 
I will align this with @romansmirnov and if necessary create a further PR to fix it.

Reply:
Sorry for not being clear enough. This would be my suggestion:
```suggestion
    public Builder processDefinitionKeys(final Long... values) {
```

Reply:
And it would apply to all other filter methods as well.